meta-mender-core/conf/layer.conf

```diff
@@ -11,11 +11,11 @@ LAYERDIR_MENDER := "${LAYERDIR}"
 BBFILES += "${LAYERDIR}/recipes-*/*/*.bb \
 	${LAYERDIR}/recipes-*/*/*.bbappend"
 
-BBFILE_COLLECTIONS += "mender"
-BBFILE_PATTERN_mender = "^${LAYERDIR}/"
-BBFILE_PRIORITY_mender = "6"
+BBFILE_COLLECTIONS += "mender-client"
```

Comment:
Reviewers Note: Also, should this be renamed or not?

Reply:
No, this should definitely be "mender", this refers to the name of the layer, not the client.